server/resp/src/main/java/org/infinispan/server/resp/commands/connection/MemoryEntrySizeUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.infinispan.server.resp.commands.connection;
+
+import org.ehcache.sizeof.SizeOf;
+import org.infinispan.container.entries.CacheEntrySizeCalculator;
+import org.infinispan.container.entries.InternalCacheEntry;
+import org.infinispan.container.entries.PrimitiveEntrySizeCalculator;
+
+public class MemoryEntrySizeUtils {
+   private static PrimitiveEntrySizeCalculator pesc = new PrimitiveEntrySizeCalculator();
+   private static CacheEntrySizeCalculator<byte[], Object> cesc = new CacheEntrySizeCalculator<byte[], Object>(
```

Comment:
Do we want to have this in core instead and update those references as well? Or is that for a new PR?

Reply:
you mean move to core and replace current computation with sizeof?

Reply:
correct.

Reply:
yeah, I guess it is worth doing all it now, let me see.

Reply:
@wburns , I changed my mind on this: I would like to go ahead with the rest of resp.json command. Can we merge this as is? I'll open an issue for adopting sizeof in the rest of the memory size code.

Reply:
Sure